src/nrc/api/tests/test_domains.py

```diff
@@ -149,3 +124,94 @@ def test_filter_attributes(self):
         self.assertEqual(
             domain.filter_attributes, ["bronorganisatie", "vertrouwelijkheid"]
         )
+
+    def test_update_domain(self):
+        """
+        test /domains PUT:
+        update existent domain
+        """
+        domain = DomainFactory.create()
+        data = {
+            "name": "someupdatedname",
+            "documentation_link": "https://example.com/doc",
+        }
+
+        domain_update_url = get_operation_url("domain_update", uuid=domain.uuid)
+
+        with requests_mock.mock() as m:
+            m.register_uri(
+                "POST", "https://new.endpoint.example.com/webhook", status_code=204
+            )
+            response = self.client.put(domain_update_url, data)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.data)
+
+        domain.refresh_from_db()
+
+        self.assertEqual(data["name"], domain.name)
+        self.assertEqual(data["documentation_link"], domain.documentation_link)
```

Comment:
Hier ben je meer benieuwd naar of het domain daadwerkelijk is geupdate in plaats van het controleren van de response.

Reply:
Bovenstaande comment klopt niet, er wordt wel degelijk vergeleken tussen het `domain` en de meegestuurde `data`